Description: Fix the docutils.utils.decode_path() function so that it accepts
 None as an argument.
Origin: http://svn.berlios.de/viewcvs/docutils?view=rev&revision=6216
Bug: http://sourceforge.net/tracker/?func=detail&aid=2923723&group_id=38414&atid=422030
Bug-Debian: http://bugs.debian.org/563087
Last-Update: 2010-01-08

Index: python-docutils-0.6/docutils/utils.py
===================================================================
--- python-docutils-0.6.orig/docutils/utils.py	2009-09-10 13:02:27.000000000 +0200
+++ python-docutils-0.6/docutils/utils.py	2010-01-08 11:40:23.000000000 +0100
@@ -338,6 +338,8 @@
     # see also http://article.gmane.org/gmane.text.docutils.user/2905
     try:
         path = path.decode(sys.getfilesystemencoding(), 'strict')
+    except AttributeError: # default value None has no decode method
+        return nodes.reprunicode(path)
     except UnicodeDecodeError:
         path = path.decode('utf-8', 'strict')
         try:
